upstream-community-operators/prometheus/prometheusoperator.0.27.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,295 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: prometheusoperator.0.27.0
+  namespace: placeholder
+  annotations:
```

Comment:
Please add the `metadata.annotations.capabilities` field, as described here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md

Reply:
@brancz For the capabilities field, I suggest `Seamless Upgrades` at the least since this operator is supporting upgrades through CSVs. How does this sound?